Detectors/ITSMFT/ITS/reconstruction/src/CookedTracker.cxx

```diff
@@ -608,38 +615,26 @@ bool CookedTracker::makeBackPropParam(TrackITS& track) const
   return true;
 }
 
-int CookedTracker::loadClusters(const std::vector<Cluster>& clusters)
+int CookedTracker::loadClusters(const std::vector<Cluster>& clusters, const o2::ITSMFT::ROFRecord& rof)
 {
   //--------------------------------------------------------------------
   // This function reads the ITSU clusters from the tree,
   // sort them, distribute over the internal tracker arrays, etc
   //--------------------------------------------------------------------
-  Int_t numOfClusters = clusters.size();
-  int nLoaded = 0;
-  mFirstCluster = &clusters.front();
+  int first = 0;
+  int number = clusters.size();
+  if (mContinuousMode) { // Load clusters from this ROFrame only, if in continuous mode
```

Comment:
I think this should not depend on the RO mode, in the triggered mode the ROF's are simply not contiguous in time, but the access method will remain the same.

Reply:
Sorry, I don't catch how this argument is related to these lines. The treatment of events crossing the ROF boundary will require loading clusters of 2 ROFs, which is anyway not covered by these lines. Here we process just one complete ROF, the ROFrecord will refer to 1st entry and N clusters in it in a same way, irrespectively of the RO mode, so the loading of the clusters for a given ROF/Trigger should be the same.

Reply:
OK, you want to ignore the ROF record in case of the "triggered RO", assuming that the whole tree event in this case will represent just one trigger. In principle this is never the case for clusters produced by the workflow and true for the output of the ``run_clus_itsSA.C`` only with ``entryPerROF = kTRUE;``, which eventually should be eliminated (once the proper triggered mode will be implemented). Then let's keep these lines till this implementation.

Reply:
@shahor02 Yes, this is what this PR is about, to make the two ITS tests work:
1) run_tests.sh, macro-driven, working in triggered mode by default.
2) run_workflows, workflow-based, working in continuous mode by default.
The tests may not work for any option combination, but at least they work with the defaults.

Eventually, we should probably reduce the number of available options, preserving the workflow
preferences.  But can we do it in a separate PR, OK ?